EventPlanner/app/src/main/java/m3/eventplanner/fragments/event/EventDetailsViewModel.java

```diff
@@ -109,16 +108,35 @@ public void onFailure(Call<Collection<GetEventDTO>> call, Throwable t) {
         });
     }
 
-    public void toggleFavourite(int accountId, int eventId) {
+    private void loadAgenda(int eventId) {
+        clientUtils.getEventService().getEventAgenda(eventId).enqueue(new Callback<Collection<GetAgendaItemDTO>>() {
+            @Override
+            public void onResponse(Call<Collection<GetAgendaItemDTO>> call, Response<Collection<GetAgendaItemDTO>> response) {
+                if (response.isSuccessful() && response.body() != null) {
```

Comment:
Add check for empty response its good practice 
!response.body().isEmpty()

Reply:
Agenda can be empty. (When the event is created and organizer didn't create any agenda items yet)